### ISSUE TITLE
disconnect -> connect does not work on PannerNode

### ISSUE THREAD

**Issue by kuviman** [orottier/web-audio-api-rs#494]
Using PannerNode, the sound is working normally unless i try to disconnect and reconnect the node to something else (or even same thing):

```rust
use web_audio_api::{
    context::{AudioContext, BaseAudioContext},
    node::{AudioBufferSourceNode, AudioNode, PannerNode},
};

fn main() {
    let audio = AudioContext::new(Default::default());
    let buffer = audio
        .decode_audio_data_sync(std::io::Cursor::new(
            std::fs::read("sound.wav").unwrap(),
        ))
        .unwrap();
    let mut source = AudioBufferSourceNode::new(&audio, Default::default());
    source.set_buffer(buffer);
    let panner = PannerNode::new(&audio, Default::default());
    source.connect(&panner);
    panner.connect(&audio.destination());

    // No sound, but is sound if these two lines are commented
    panner.disconnect();
    panner.connect(&audio.destination());

    source.start_at_with_offset(0.0, 0.0);
    std::thread::sleep(std::time::Duration::from_secs(1));
}
```

**Comment by orottier** [comment]
Hi @kuviman, thanks for the report.
We literally yesterday fixed a bug where `disconnect` would also remove incoming connections (instead of only the outgoing ones): #493
I think your code is also affected by that bug. You could try if `source.connect(&panner)` brings back the sound.
The fix is on master already and I intend to release a new version to crates.io later today.

**Comment by orottier** [comment]
I have just released version `v1.0.0-rc.5` which should solve your problem. Please let us know if you still run into issues!
